programmers/체육복.js

```diff
@@ -0,0 +1,14 @@
+function solution(n, lost, reserve) {
+  let duplicated = lost.filter((val) => reserve.includes(val));
+  lost = lost.filter((val) => !duplicated.includes(val));
+  reserve = reserve.filter((val) => !duplicated.includes(val));
+  lost = lost.filter((lostVal) => {
+    let reserveIndex = reserve.findIndex((reserveVal) => (lostVal === reserveVal + 1 || lostVal === reserveVal - 1));
```

Comment:
변수명 뒤에 ```val``` ```data``` ```arr``` ```str``` 등과 같은 이름은 사용하지 않는 것이 좋습니다.
정말 어쩔 수 없는 특별한 경우에만 사용하는게 좋아요.
```suggestion
    const reserveIndex = reserve.findIndex((reserveVal) => (lostVal === reserveVal + 1 || lostVal === reserveVal - 1));
```